src/test/java/com/instana/operator/customresource/InstanaAgentSpecDeserializeTest.java

```diff
@@ -55,21 +55,14 @@ void testDefaultValues() throws IOException {
     assertThat(spec.isAgentRbacCreate(), equalTo(Boolean.parseBoolean(DEFAULT_AGENT_RBAC_CREATE)));
```

Comment:
Is there a cleaner way we can do these assertions (e.g. a spec matcher)? Not a blocker.

Reply:
Maybe?